lib/env.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import * as lambda from "@aws-cdk/aws-lambda";
+
+
+export const enableDDTracingEnvVar = "DD_TRACE_ENABLED";
+export const injectLogContextEnvVar = "DD_LOG_INJECTION";
+
+export const defaultEnvVar = {
+  addLayers: true,
+  enableDDTracing: true,
+  injectLogContext: true,
+};
+
+export function applyEnvVariables(
+    lambdas: lambda.Function[],
+    enableDDTracing: boolean,
+    injectLogContext: boolean,
+) {
+  lambdas.forEach((l) => {
+    l.addEnvironment(enableDDTracingEnvVar, enableDDTracing.toString().toLowerCase());
```

Comment:
This is fine for a small piece of code, but for larger code we generally prefer full length variable names, eg 'lambda'.

Reply:
I changed this from `l` to `lam` as there is a naming conflict if we used `lambda`. The lambda CDK library is imported under the `lambda` alias and is best practice. Let me know if this works for you or if you have another suggestion.